config/detekt-baseline.xml

```diff
@@ -210,6 +210,7 @@
     <ID>UndocumentedPublicClass:SessionUseCases.kt$SessionUseCases$ExitFullScreenUseCase</ID>
     <ID>UndocumentedPublicClass:SessionUseCases.kt$SessionUseCases$GoBackUseCase</ID>
     <ID>UndocumentedPublicClass:SessionUseCases.kt$SessionUseCases$GoForwardUseCase</ID>
+    <ID>UndocumentedPublicClass:SessionUseCases.kt$SessionUseCases$GoToHistoryIndexUseCase</ID>
```

Comment:
You could always add the docs too. :)

These detekt warnings existed before we modified our lint configuration to catch it. They were ignored as part of a baseline so that you don't have to add docs for all the code that didn't already exist, but just for the new methods.we add.